core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/split/SplitOpDesc.scala

```diff
@@ -48,13 +48,13 @@ class SplitOpDesc extends LogicalOp {
 
   override def operatorInfo: OperatorInfo = {
     OperatorInfo(
-      userFriendlyName = "Training/Testing Split",
-      operatorDescription = "Split training and testing data to two different ports",
+      userFriendlyName = "Split",
+      operatorDescription = "Split data to two different ports",
       operatorGroupName = OperatorGroupConstants.MACHINE_LEARNING_GROUP,
```

Comment:
Move it to other group. Let's not make it an ML specific operator. It should be a general operation

Reply:
moved to utilities group